xonsh/prompt/vc.py

```diff
@@ -9,6 +9,7 @@
 import threading
 import subprocess
 import re
+from pathlib import Path
```

Comment:
Please import `pathlib` and then use `pathlib.Path` below so that this gets the benefit of amalgamation

Reply:
Reading https://github.com/xonsh/amalgamate#imports, it seems that I could also just change the `from ... import` statement as `import pathlib.Path as Path` to get the advantage of the lazy import. Can I do that? Like that the rest of the code is not modified by this "amalgamate.py" particularity and stays as if it were written with the standard `from ... import` shown in https://docs.python.org/3/library/pathlib.html#basic-use.

Reply:
Yep! You could use that style of import. But to get the import advantage, you'd have to change other modules to use exactly the same import line.  Currently a lot of them just use `import pathlib`.  Your choice, but my advice would be to conform to the imports for this PR and then if you wanted to do a PR that changed all of the pathlib imports, you could put that in separately.

Reply:
Really either way though!

Reply:
I understand. Let's go for `import pathlib` then!